lib/pacemaker/pcmk_rule.c

```diff
@@ -111,6 +111,8 @@ init_rule_check(pcmk__output_t *out, xmlNodePtr input, crm_time_t *date,
     return pcmk_rc_ok;
 }
 
+#define XPATH_NODE_RULE "//" XML_TAG_RULE "[@id='%s']"
```

Comment:
Some of the other xpath components have constants e.g. XML_ATTR_ID, XML_EXPR_ATTR_OPERATION

Reply:
I must have stopped checking after I saw that `date_expression` didn't have a constant